lib/ansible/modules/messaging/rabbitmq_policy.py

```diff
@@ -149,6 +168,7 @@ def main():
         tags=dict(type='dict', required=True),
         priority=dict(default='0'),
         node=dict(default='rabbit'),
+        extra_ctl_paths=dict(default=list(), type='list'),
```

Comment:
This is the same as above - it's used in 4 files, and not appropriate or used in 4 other rabbit files. Do you still thing it should be common in light of this ? 

Thanks!